src/tennisSimulator/MatchSimulator.java

```diff
@@ -0,0 +1,24 @@
+package tennisSimulator;
+
+public class MatchSimulator {
+
+    public void firstServe(Player firstPlayer, Player secondPLayer)  {
```

Comment:
I have changed a concept of this class a little. Since we know that after we start simulating match players won`t change, so I introduced two Player fields and removed them from methods input`s.